beacon-chain/sync/validate_blob.go

```diff
@@ -159,35 +164,39 @@ func (s *Service) validateBlobPostSeenParent(ctx context.Context, sBlob *eth.Sig
 	if err != nil {
 		return pubsub.ValidationIgnore, err
 	}
-	if err := verifyBlobSignature(parentState, sBlob); err != nil {
-		log.WithError(err).WithFields(blobFields(blob)).Debug("Failed to verify blob signature")
-		return pubsub.ValidationReject, err
-	}
+	// TODO: replace this with verification routine
+	/*
+		if err := verifyBlobSignature(parentState, blob); err != nil {
+			log.WithError(err).WithFields(blobFields(blob)).Debug("Failed to verify blob signature")
+			return pubsub.ValidationReject, err
+		}
+	*/
 
 	// [IGNORE] The sidecar is the only sidecar with valid signature received for the tuple (sidecar.block_root, sidecar.index).
-	if s.hasSeenBlobIndex(blob.BlockRoot, blob.Index) {
+	if s.hasSeenBlobIndex(blob.BlockRootSlice(), blob.Index) {
 		return pubsub.ValidationIgnore, nil
 	}
 
 	// [REJECT] The sidecar is proposed by the expected proposer_index for the block's slot in the context of the current shuffling (defined by block_parent_root/slot)
-	parentState, err = transition.ProcessSlotsUsingNextSlotCache(ctx, parentState, parentRoot[:], blob.Slot)
+	parentState, err = transition.ProcessSlotsUsingNextSlotCache(ctx, parentState, parentRoot[:], blob.Slot())
```

Comment:
Why are we grabbing a state to do this, can't we use the proposer index cache instead?

Reply:
Agreed, but this line only changed because I updated the slot accessor - let's resolve that in the upcoming verification PR: https://github.com/prysmaticlabs/prysm/pull/13197

Reply:
I can handle this check in my gossip PR as well